CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ https://docs.docker.com/engine/deprecated/ where you can find target removal dat
 + Adds a new compose file version 3.6 [docker/cli#808](https://github.com/docker/cli/pull/808)
 - Fix issue of filter in `docker ps` where `health=starting` returns nothing [moby/moby#35940](https://github.com/moby/moby/pull/35940)
 + Improve presentation of published port ranges [docker/cli#581](https://github.com/docker/cli/pull/581)
+* Bump Go to 1.9.3 [docker/cli#827](https://github.com/docker/cli/pull/827)
+- Fix broken Kubernetes stack flags [docker/cli#831](https://github.com/docker/cli/pull/831)
+* Annotate "stack" commands to be "swarm" and "kubernetes" [docker/cli#804](https://github.com/docker/cli/pull/804)
```

Comment:
This actually isn't new and was already in 18.01? But I guess you cherry-picked it in specially?

Reply:
Looks like the [fix](https://github.com/docker/cli/pull/804) is not part of 18.01 and the fix also missed the code freeze for 18.02, but now it is backported for 18.02 via #409.

Reply:
fyi #409 is now merged